lib/quizBrain.dart

```diff
@@ -1,8 +1,19 @@
 import 'questions.dart';
 
+// review this code
+
+// Overall review: This is one solid code. The class accomplishes what it sets outs to do, the methods implemented are taking one responsibility which is really good
+//very well written class!
 class QuizBrain {
+  //SUGGESTION: maybe change the variable name to a more mnemonic word like _currentQuestionIndex
   int _tracker = 0;
 
+  //SUGGESTION: instead of interacting with _question maybe use a void function that lets you add questions 
+  //This will make your code scalable and easier to manage. You can also add a method that removes a question if it is desired to remove a question
+  //something like this
+  //void addQuestion(String questionName){
+  //  _questions.add(Question(questionName));
+  //}
   final List<Question> _questions = [

```

Comment:
Instead of adding Questions manually to the list I suggest having a method that adds a question when you want for scalability

void addQuestion(String questionAsked, bool answer){
    _question.add(questionAsked, answer);
}